src/scss/blocks/sign-in-button.scss

```
@@ -0,0 +1,27 @@
+.sign-in-button__img{
+    width: 32px;
+    height: 32px;
+}
+.sign-in-button__circle{
+    background: #FFFFFF;
+    border-radius: 16px;
+    width: 32px;
+    height: 32px;
+}
+.sign-in-button__item-name{
+    font-weight: 400;
+    font-size: 15px;
+    color: #F5F5F5;
+    @include media-xl-max{
+        display: none;
+    }
+    @include media-lg-max{
+        display: none;
+    }
+    @include media-md-max{
+        display: none;
+    }
+    @include media-sm-max{
+        display: none;
+    }
+}
```

логику с display: none; нужно подправить, зачем на все ендпоинты один и тот же css

---

@ysv-a  А как можно скрыть этот элемент во всех состояниях экрана меньше чем medai-xl-max? Я вроде по лекциям искал, не нашел подобного. Вот такую реализацию только замутил

---

это значит что если экран будет меньше 1260px, то применить правила
```suggestion
.sign-in-button__item-name{
    font-weight: 400;
    font-size: 15px;
    color: #F5F5F5;
    @include media-xl-max{
        display: none;
    }
}
```

---

готово